app/models/detailModel.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+namespace Application\Models\DetailModel;
+
+// use Application\Core\Database\Database\DatabaseConnection; still in main controler
+
+class DetailModel
```

Comment:
Il faut que tu renome ce file en DetailModel (aussi, pour les models par convention quand c'est dans le folder Model tu as pas besoin de mettre Model à la fin du nom de la class)

Reply:
Ok i did it 


<img width="237" alt="Capture d’écran 2022-07-11 à 11 14 19" src="https://user-images.githubusercontent.com/18627668/178240890-bb4278f3-517c-4a82-8eac-adcb8bdea3ca.png">

for "pas besoin de mettre Model à la fin du nom de la class)" as i already have a class names Detail j Juste renamed the first One witch is useles
<img width="531" alt="Capture d’écran 2022-07-11 à 11 58 23" src="https://user-images.githubusercontent.com/18627668/178241342-f59a420b-c1df-45b6-9d54-9407173d104e.png">
s